src/zabapgit_dot_abapgit.prog.abap

```diff
@@ -103,7 +108,7 @@ CLASS lcl_dot_abapgit IMPLEMENTATION.
     DATA: ls_data TYPE ty_dot_abapgit.
 
 
-    ls_data-master_language = iv_master_language.
+    ls_data-master_language = sy-langu.
```

Comment:
mmm. Well, I guess should be no problem for now. If someone face it they will open a specific issue with a specific case.